packages/svelte/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "remix-router-svelte",
+  "version": "0.3.0",
+  "description": "A Svelte UI layer for nested + data-driven routing via @remix-run/router",
+  "author": "aust.crim@gmail.com",
+  "license": "MIT",
+  "main": "dist/remix-router-svelte.umd.js",
+  "module": "dist/remix-router-svelte.es.js",
+  "types": "dist/src/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/remix-router-svelte.es.js",
+      "require": "./dist/remix-router-svelte.umd.js"
+    }
+  },
+  "sideEffects": false,
+  "homepage": "https://github.com/brophdawg11/remix-routers#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brophdawg11/remix-routers.git",
+    "directory": "packages/svelte"
+  },
+  "bugs": {
+    "url": "https://github.com/brophdawg11/remix-routers/issues"
+  },
+  "keywords": [
+    "remix",
+    "remix-run",
+    "router",
+    "svelte"
+  ],
+  "files": [
+    "dist/**"
+  ],
+  "scripts": {
+    "build": "vite build && tsc -b",
+    "clean": "git clean -fX .",
+    "dev": "vite dev",
+    "integration:open": "cypress open --config-file ./../../cypress.config.ts",
+    "integration:start": "vite dev --mode=production --port 3000",
+    "integration:test": "cypress run --config-file ./../../cypress.config.ts",
+    "integration": "start-server-and-test integration:start http-get://localhost:3000/ integration:test",
+    "prepublishOnly": "npm run build"
+  },
+  "dependencies": {
+    "@remix-run/router": "0.1.0",
```

Comment:
I've updated `main` to the latest version here so you'll want to update this to match what's in react/vue - currently `0.2.0-pre.2`